src/HotChocolate/Core/src/Subscriptions.Nats/HotChocolate.Subscriptions.Nats.csproj

```diff
@@ -0,0 +1,21 @@
+<Project Sdk="Microsoft.NET.Sdk">
+
+    <PropertyGroup>
+        <TargetFramework>net6.0</TargetFramework>
+        <ImplicitUsings>enable</ImplicitUsings>
+        <Nullable>enable</Nullable>
+        <TargetFrameworks />
+    </PropertyGroup>
+
+    <ItemGroup>
+      <ProjectReference Include="..\Execution\HotChocolate.Execution.csproj" />
+      <ProjectReference Include="..\Subscriptions\HotChocolate.Subscriptions.csproj" />
+    </ItemGroup>
+
+    <ItemGroup>
+      <PackageReference Include="AlterNats.Hosting" Version="1.0.5" />
```

Comment:
Is the reason to use this client and not the offical one performance?

Reply:
Yeah, the official package is really awful -- it's a straight port of the Go version, and doesn't use any of the normal dotnet idioms. AlterNats is zero-alloc and very modern, has more functionality, and is following all the idioms with async that you'd want to see.